app/javascript/shared/components/ChatForm.vue

```diff
@@ -3,14 +3,28 @@
     <form @submit.prevent="onSubmit">
       <div v-for="item in items" :key="item.key" class="form-block">
         <label>{{ item.label }}</label>
-        <input
-          v-if="item.type === 'email' || item.type === 'text'"
-          v-model="formValues[item.name]"
-          :type="item.type"
-          :name="item.name"
-          :placeholder="item.placeholder"
-          :disabled="!!submittedValues.length"
-        />
+        <div v-if="item.type === 'email'">
```

Comment:
We would need to do the validation on all input text elements available in chatwoot
`text`, `email`

So let's remove the specific condition added.